README.md

```diff
@@ -101,6 +101,39 @@ The notification will be sent from your Facebook page, whose page token you have
 
 ![Laravel Facebook Notification Example](https://cloud.githubusercontent.com/assets/1915268/17666125/58d6b66c-631c-11e6-9380-0400832b2e48.png)
 
+#### Message Examples
+##### Basic Text Message
+Send a basic text message to a user
+```php
+return FacebookMessage::create('You have just paid your monthly fee! Thanks')
+->to($this->user->fb_messenger_id);
+```
+##### Attachment Message
+Send a file attachment to a user (Example is sending a pdf invoice)
```

Comment:
Not sure if the below attachment is of PDF or an image? Because it says `AttachmentType::IMAGE` with a URL. It doesn't seem PDF.